src/test/java/seedu/address/testutil/TypicalClients.java

```diff
@@ -88,14 +87,14 @@ public static TbmManager getTypicalTbmManager() {
     }
 
     public static List<Client> getTypicalClients() {
-        Client alice = new ClientBuilder(TypicalClients.ALICE).build();
-        Client benson = new ClientBuilder(TypicalClients.BENSON).build();
-        Client carl = new ClientBuilder(TypicalClients.CARL).build();
-        Client daniel = new ClientBuilder(TypicalClients.DANIEL).build();
-        Client elle = new ClientBuilder(TypicalClients.ELLE).build();
-        Client fiona = new ClientBuilder(TypicalClients.FIONA).build();
-        Client george = new ClientBuilder(TypicalClients.GEORGE).build();
-        return new ArrayList<>(Arrays.asList(alice, benson, carl, daniel, elle, fiona, george));
+        Client alice = new ClientBuilder(ALICE).build();
+        Client benson = new ClientBuilder(BENSON).build();
+        Client carl = new ClientBuilder(CARL).build();
+        Client daniel = new ClientBuilder(DANIEL).build();
+        Client elle = new ClientBuilder(ELLE).build();
+        Client fiona = new ClientBuilder(FIONA).build();
+        Client george = new ClientBuilder(GEORGE).build();
+        return (Arrays.asList(alice, benson, carl, daniel, elle, fiona, george));
```

Comment:
```suggestion
        return Arrays.asList(alice, benson, carl, daniel, elle, fiona, george);
```

No need for extra brackets

Reply:
No idea how you're so attentive :goat: